NSS-Day-Cohort-49/tabloid-cli-spiny-crabs
# Mat blog tags

#### Changes Made
1. Modified methods in blogRepository to get post information when getting author by ID
1. Modified blog detail manager to have methods for adding tags to blogs, viewing blog tags, removing blog tags, and viewing posts in a blog
?
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout mat-blogTags
    ```
2. Open a new Terminal tab (?T) and navigate to the server directory.
3. Test app functionality.
    > Navigate to blog details
    > Add a tag for a blog
    > View to view a blog's details & tags
    > remove a tag from a blog
    > Select view posts to see all posts related to chosen blog
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
